src/index.html

```diff
@@ -33,7 +33,7 @@
 	</div>
 
 </body>
-
+<script src="/iris/http/config.js"></script>
```

Comment:
Instead of including code at render, this should be handled _within_ the app itself (ie at event `PUSHER_CONNECTED`)